<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admdel</class>
 <widget class="QDialog" name="admdel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image/srcc.qrc">
    <normaloff>:/img/logo1.jpg</normaloff>:/img/logo1.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qconicalgradient(cx:0.548913, cy:1, angle:0, stop:0 rgba(0, 0, 103, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLineEdit" name="product_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Product_cat">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Cat:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Price">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Price:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QLabel" name="product_name">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Manufacturer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Manufacturer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cat_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Product_desc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Dsc:</string>
   </property>
  </widget>
  <widget class="QLabel" name="quantity">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Qty:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qty_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="description">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Manufact_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_invent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-70</y>
     <width>151</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/del.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image/srcc.qrc"/>
 </resources>
 <connections/>
</ui>
